Clients/src/presentation/components/Table/index.tsx

```
@@ -36,6 +38,11 @@ interface TableData {
   rows: RowData[];
 }
 
+type TableRow = {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'TableRow'. Consider to delete it or rename it.

'TableRow' is defined here:


(lint/suspicious/noRedeclare)